packages/jest-runner/src/runTest.ts

```diff
@@ -312,8 +321,12 @@ async function runTestInternal(
         sendMessageToJest,
       );
     } catch (error: any) {
-      // Access stack before uninstalling sourcemaps
-      error.stack;
+      // Access all stacks before uninstalling sourcemaps
+      let e = error;
+      while (typeof e === 'object' && 'stack' in e) {
+        e.stack;
+        e = e?.cause ?? {};
```

Comment:
do we need the `??` part? the `while` condition does a `typeof` which should be fine? Or is it to guard against `null`?

Reply:
@SimenB Pretty much, yeah.

Maybe it's better to add the `null` protection explicitly to the `while` condition; this way, it would also protect against a `null` been thrown for some reason.